src/scripts/add_user.py

```diff
@@ -0,0 +1,22 @@
+from getpass import getpass
```

Comment:
we should probably adapt this later on into an interface for users to use.

Reply:
Unsure. The software is intended for admins. So then how to block any undesired party from having access to that registration page?